### PR TITLE
DataBrew Content

### PR DESCRIPTION
**Pull request by AhmedsZamzam**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: workshop/content/DataBrew/01_Setting_up.en.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+title = "Setting up DataBrew"
+date = 2020-01-30T20:12:40+01:00
+weight = 01
+chapter = true
+pre = "<b>0.1. </b>"
++++
+
+In this section we will setup an AWS Glue DataBrew project which is an interactive data preparation workspace in DataBrew. During this process we will create the following:
+
+- **Dataset:** rows or records that are divided into columns or fields.
```
Comment (sinanerdem) [inline workshop/content/DataBrew/01_Setting_up.en.md]: I am not very sure about this definition. In the docs it says ["In AWS Glue DataBrew, a dataset represents data that's stored elsewhere and details on how DataBrew can connect to that data."](https://docs.aws.amazon.com/databrew/latest/dg/datasets.html)

File: workshop/content/DataBrew/01_Setting_up.en.md [hunk 2]
```diff
@@ -8,28 +8,29 @@ pre = "<b>0.1. </b>"
 
 In this section we will setup an AWS Glue DataBrew project which is an interactive data preparation workspace in DataBrew. During this process we will create the following:
 
-- **Dataset:** rows or records that are divided into columns or fields.
```
Comment (sinanerdem) [inline workshop/content/DataBrew/01_Setting_up.en.md]: Used the definition from the [docs](https://docs.aws.amazon.com/databrew/latest/dg/datasets.html)

**Review by AhmedsZamzam — COMMENTED** [review]
Agree with all changes. I cannot approve it, since I am the author of the PR.

**Comment by sinanerdem** [comment]
Only @ahmedhany or @BastLeblanc can approve it I think.

**Review by ahmedhany — APPROVED** [review]
Reviewed by Sinan

**Pull request merged**